_includes/url/url-home.html

```diff
@@ -0,0 +1 @@
+href="{% include url/url-base-path.html %}"
```

Comment:
These url includes seem really weird? Is this some jekyll standard?

Reply:
Not a standard that I know of. See above comments about URL abstraction. If you want me to hard-code and repeat these URLs I'm ok with that. Just let me know.

Reply:
Will take a stab at using Jekyll variables instead in next pass.